include/pmacc/particles/ParticlesBase.hpp

```diff
@@ -172,6 +172,11 @@ class ParticlesBase : public SimulationFieldHelper<T_MappingDesc>
     /** copy gard particles to intermediate exchange buffer
```

Comment:
Typo: `gard ` instead of `guard`

Reply:
I made this typo very often, as soon as this PR is merged I will fix is globally

Reply:
issue #2643